schemas/snuba-lw-deletions.v1.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "lw_delete",
+  "type": "object",
+  "properties": {
+    "storage_name": {
+      "type": "string"
+    },
+    "rows_to_delete": {
+      "type": "integer"
+    },
+    "conditions": {
+      "type": "object",
+      "oneOf": [{ "$ref": "#/definitions/SearchIssuesConditions" }]
```

Comment:
why do we have oneOf, instead of the ref directly?

Reply:
so that later you can just add another definition for a different storage